ben02/mod_ben02.F

```diff
@@ -98,6 +107,20 @@ module mod_ben02
 c
       integer ::
      .  ntda           ! accumulation number
+c
```

Comment:
New functionality added in mod_ben02 (dfl, albw, alb, rnfins, rnfres, nrfets) ?

Reply:
Sorry, I wanted to point to the lines following line 110 ...

Reply:
My comment https://github.com/NorESMhub/BLOM/pull/61#discussion_r560225748 is related to this. No new functionality, just restructuring.